server/db/testDB/build.js

```diff
@@ -0,0 +1,17 @@
+const dbConnection = require("../config/connection");
+const fs = require("fs");
```

Comment:
we require the core modules before your local modules 
```suggestion
const fs = require("fs");
const dbConnection = require("../config/connection");
```